--- a/pkg/authn/k8schain/k8schain.go	2019-07-30 15:46:30.407646356 +0100
+++ b/pkg/authn/k8schain/k8schain.go	2019-07-30 15:47:41.947940641 +0100
@@ -26,7 +26,6 @@

 	// Credential providers
 	_ "k8s.io/kubernetes/pkg/credentialprovider/aws"
-	_ "k8s.io/kubernetes/pkg/credentialprovider/azure"
 	_ "k8s.io/kubernetes/pkg/credentialprovider/gcp"
 	// TODO(mattmoor): This doesn't seem to build, figure out why `dep ensure`
 	// is not working and add constraints.
