--- a/vault/testing.go	2019-12-02 11:21:18.670447489 +0000
+++ b/vault/testing.go	2019-12-02 11:23:22.494887299 +0000
@@ -43,7 +43,6 @@
 	"github.com/hashicorp/vault/logical"
 	"github.com/hashicorp/vault/logical/framework"
 	"github.com/hashicorp/vault/physical"
-	dbMysql "github.com/hashicorp/vault/plugins/database/mysql"
 	dbPostgres "github.com/hashicorp/vault/plugins/database/postgresql"
 	testing "github.com/mitchellh/go-testing-interface"

@@ -1513,7 +1512,6 @@
 func NewMockBuiltinRegistry() *mockBuiltinRegistry {
 	return &mockBuiltinRegistry{
 		forTesting: map[string]consts.PluginType{
-			"mysql-database-plugin":      consts.PluginTypeDatabase,
 			"postgresql-database-plugin": consts.PluginTypeDatabase,
 		},
 	}
@@ -1531,10 +1529,7 @@
 	if pluginType != testPluginType {
 		return nil, false
 	}
-	if name == "postgresql-database-plugin" {
-		return dbPostgres.New, true
-	}
-	return dbMysql.New(dbMysql.MetadataLen, dbMysql.MetadataLen, dbMysql.UsernameLen), true
+  return dbPostgres.New, true
 }

 // Keys only supports getting a realistic list of the keys for database plugins.
@@ -1547,10 +1542,6 @@
 		The registry isn't directly used because it causes import cycles.
 	*/
 	return []string{
-		"mysql-database-plugin",
-		"mysql-aurora-database-plugin",
-		"mysql-rds-database-plugin",
-		"mysql-legacy-database-plugin",
 		"postgresql-database-plugin",
 		"mssql-database-plugin",
 		"cassandra-database-plugin",
